packages/auth-compat/src/platform.ts

```diff
@@ -51,7 +51,7 @@ function _isHttpOrHttps(): boolean {
  */
 export function _isAndroidOrIosCordovaScheme(ua: string = getUA()): boolean {
   return !!(
-    (_getCurrentScheme() === 'file:' || _getCurrentScheme() === 'ionic:') &&
+    (_getCurrentScheme() === 'file:' || _getCurrentScheme() === 'ionic:' || _getCurrentScheme() === 'capacitor:') &&
```

Comment:
Please run `yarn format` in the root of the repo (I think this will cause the line to wrap)

Reply:
Done :)

Reply:
Done